class-wc-calypso-bridge-frontend-free-trial.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Class WC_Calypso_Bridge_Frontend_Free_Trial.
+ *
+ * @since   1.9.16
+ * @version 1.9.16
+ *
+ * Handles Free Trial frontend.
+ */
+class WC_Calypso_Bridge_Frontend_Free_Trial  {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+	
+	public function __construct(){
+		add_action('init', function() {
+			if (current_user_can('manage_woocommerce')) {
+				add_action('wp_footer', array($this, 'add_plan_picker_banner'), 10);
+			}
+		});
+
+		add_action('wp_enqueue_scripts', array($this, 'add_styles'));
+	}
+
+	/**
+	 * @return void
+	 */
+	public function add_plan_picker_banner()
+	{
+		$link = 'https://wordpress.com/';
```

Comment:
The link should point to the correct plan URL - As discussed with @xristos3490 the most robust way is the following

```php
$site_slug = ( new \Automattic\Jetpack\Status() )->get_site_suffix();
$plan_url  = 'https://wordpress.com/plans/' . $site_slug;
```

Reply:
I see that there is a function in this PR `get_action_url` that is already doing the same

Reply:
Thank you @PanosSynetos 👍 

I wasn't aware of `get_action_url` when I committed this change. I'll update it soon.